### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by pavlobu** [pavlobu/deskreen#88] [General]
Hi all. 👋 This thread is for general discussion. You can discuss general topics here, for example how did you find out about Deskreen? Or send cheers and thanks to anyone in Deskreen's community members. 🎉

**Comment by ntap-leeray** [comment]
How, if at all possible, to get Deskreen to work on a Chromebook or ChromeOS devices?

**Reply by pavlobu** [reply]
you can download .AppImage from deskreen.com or from releases page on github

**Comment by gcleaves** [comment]
Thanks for Release v1.0.9, Deskreen now works on Windows 10 host.

**Comment by dhrm1k** [comment]
You are doing great work brother. Thanks for such a great service.

**Comment by vk6996** [comment]
May know u who is this?

**Comment by wdalto** [comment]
Dear pavlobu

Would it be possible to compile Descreen code for 32-bit linux architecture? I would like to reuse old equipment with a second screen configuration that would otherwise end up in the trash. It would be an attitude in favor of sustainability.

Thank you very much,

**Comment by Stephane-Lpt** [comment]
Hello guys, 
Is it possible to attribute a static ip adress for deskreen ? I would like to make a script that launch everything to be ready to work in 30sec. Or is there any file where the dynamic ip adress is written ?
Thank you in advance and congratulations on making this good software

**Comment by BarleyXu** [comment]
Hi, how can I run deskreen app on Android platform?

**Comment by mmoalem** [comment]
Hi all - I have just found this software and it literally made my day, finally, after trying every workaround I could find on the net, I can finally make proper use of my new AR glasses (Xreal) - it was a real pain to find a way to anchor virtual screens from an AMD windows PC with crappy software from Xreal and workaround that either do not work on AMD or android or windows etc...
anyway - one this that is a fuff with deskreen is that you have to go through 4 or 5 choices everytime you want to connect a device (choose app or screen, which screen, allow device etc) and I connect to the instances (three virtual screen)... and also inputting the long URL inside the AR glasses is a pain due to the hardware setup... SO the question is can presets be set (same URL, choce of screen etc) so we need to click once to start it on the PC and bookmark the URL in the browser in the AR glasses?

**Comment by macooper59** [comment]
Hello Everyone,

This is a great application thank you for creating this. I have a question or 2 on this. I see a lot of Videos but none of them tell me if I have to install the software on both devices. Do I install on the device I want to use as my main system and I am almost certain on the system (my laptop) to use it's screen. Also I would think the dummy hdmi plug goes in my laptop so I can share it's screen. Is this correct?

Thank you again,
Michael aka: nexusguy59
